README.md

```diff
@@ -17,6 +17,11 @@ In most cases, the PoC should
 * run on a variety of hardware configurations including virtual and multiple
   vendors' devices
 
+# I Want To Contribute!
+
+This project welcomes contributions and suggestions.  We are happy to have the Community involved via submission of **Issues and Pull Requests** (with substantive content or even just fixes). We are hoping for the documents, test framework, etc. to become a community process with active engagement.  PRs can be reviewed by by any number of people, and a maintainer may accept.
```

Comment:
s/by by/by/

Also, please wrap on 80 columns.

Reply:
Sure, some kind of cut and paste goof, thanks for catching.